sesman/chansrv/clipboard.c

```diff
@@ -158,7 +158,7 @@ x-special/gnome-copied-files
 
 
 */
-
+#include <../config.h>
```

Comment:
`#include <...>` is mostly used for system headers or external libraries, you should better use `#include "..."`. This could also fix the modifications you added in `Makefile.am`